qemu/tests/cfg/hotplug_mem_stress.cfg

```diff
@@ -0,0 +1,27 @@
+- hotplug_mem_stress:
```

Comment:
How about indicate `ng`,  otherwise from the case name, it is easy to get confused with `hotplug_memory..stress`.

Same with 
```
stress_dir = "/tmp/stress-ng"
...
    compile_stress = "cd ${stress_dir} && make -j4"
    run_stress = "cd ${stress_dir} && ./stress-ng %s"
    stress_args = "--page-in -r 4 -t 60s;--malloc 4 -t 60s;--mmap 4 -t 60s;--cpu 5 --vm 9 --vm-bytes 40% --iomix 6 --iomix-bytes 40% --memcpy 2 --cache-prefetch -t 1200s"
```

Reply:
Sure, will add 'ng' suffix.